workspace/workflows/jobs.py

```diff
@@ -0,0 +1,220 @@
+import argparse
+import json
+import os
+import warnings
+
+import requests
+
+from ebmbot import settings
+from workspace.utils.blocks import (
+    get_basic_header_and_text_blocks,
+    get_header_block,
+    get_text_block,
+)
+
+
+def load_config():
+    path = settings.APPLICATION_ROOT / "workspace" / "workflows" / "config.json"
+    return json.loads(path.read_text())
+
+
+def match_shorthand(org):
+    return load_config()["shorthands"].get(org, org)
+
+
+def get_organization_repos(org):
+    return load_config()["repos"][org]
+
+
+def alias_status(status):
+    return load_config()["status_aliases"].get(status, status)
+
+
+TOKEN = os.environ["DATA_TEAM_GITHUB_API_TOKEN"]  # requires "read:project" and "repo"
+
+
+# TODO:Make this a shared function
+def get_api_result_as_json(url: str, params: dict) -> dict:  #  pragma: no cover
+    headers = {"Authorization": f"Bearer {TOKEN}"}
+    prms = {**params, **{"format": "json"}}
+    response = requests.get(url, headers=headers, params=prms)
+    response.raise_for_status()
+    return response.json()
+
+
+class WorkflowReporter:
```

Comment:
This WorkflowReporter is doing quite a lot of things. Could you add some comments (at least a main docstring) to say what it's doing - mostly for future us when we're trying to remember what we did and why :slightly_smiling_face: